static/src/stylesheets/module/email/_form.scss

```diff
@@ -7,6 +7,11 @@
     // adjustments, off grid
     width: calc(100% + 8px);
     margin-left: -4px;
+
+    @include mq($from: mobileLandscape, $until: tablet) {
+        margin-right: 50%;
+        width: calc(50% + 8px);
```

Comment:
Fair enough, I found a bunch of stuff about the long-gone css expressions, and some anecdotal stuff around calc but no solid numbers. I think I'll stick with `calc` as the alternatives to using it are fairly loop-the-loop layout wise but I am going to raise myself an issue to look into this a bit more and swap it out if need be. I'll let you know if I find anything, shout if you come across anything yourself.